<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventPage</class>
 <widget class="QWidget" name="EventPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_9">
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Trigger</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="3" column="0">
         <widget class="QRadioButton" name="radio_autorun">
          <property name="text">
           <string>Autorun</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="radio_event_touch">
          <property name="text">
           <string>Event Touch</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="radio_player_touch">
          <property name="text">
           <string>Player Touch</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="radio_action_button">
          <property name="text">
           <string>Action Button</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QRadioButton" name="radio_parallel_process">
          <property name="text">
           <string>Parallel Process</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Autonomous Movement</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Type:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="button_move_route">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Move Route</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="combo_move_type">
          <item>
           <property name="text">
            <string>Fixed</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Random</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Approach</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Custom</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Speed:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Freq:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="combo_speed">
          <item>
           <property name="text">
            <string>1: Slowest</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2: Slower</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3: Slow</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4: Fast</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5: Faster</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6: Fastest</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="combo_freq">
          <item>
           <property name="text">
            <string>1: Lowest</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2: Lower</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3: Low</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4: High</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5: Higher</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6: Highest</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Graphics</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Conditions</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QCheckBox" name="check_switch2">
          <property name="text">
           <string>Switch</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="combo_switch2"/>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="check_switch1">
          <property name="text">
           <string>Switch</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="check_self_switch">
          <property name="text">
           <string>Self Switch</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="combo_variable"/>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="spin_variable">
          <property name="minimum">
           <number>-9999999</number>
          </property>
          <property name="maximum">
           <number>9999999</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="check_variable">
          <property name="text">
           <string>Variable</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="combo_switch1"/>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>or above</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>is ON</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>is ON</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>is</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="combo_self_switch">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>50</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>A</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>B</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>C</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>D</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>is ON</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2" rowspan="3">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Event List</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_10">
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="button_add_command">
        <property name="text">
         <string>Add Command</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="EventList" name="eventList"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QCheckBox" name="check_move_animation">
          <property name="text">
           <string>Move Animation</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="check_through">
          <property name="text">
           <string>Through</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="check_stop_animation">
          <property name="text">
           <string>Stop Animation</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="check_direction_fix">
          <property name="text">
           <string>Direction Fix</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="check_always_on_top">
          <property name="text">
           <string>Always on Top</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EventList</class>
   <extends>QListView</extends>
   <header>event_dialogs/eventlist.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>check_switch2</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_switch2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_switch1</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_switch1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_variable</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_variable</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_self_switch</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_self_switch</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_variable</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_variable</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
